Example/RxGesture/ViewController.swift

```diff
@@ -39,18 +39,24 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        let steps: [Step] = [
+        navigationController?.navigationBar.shadowImage = UIImage()
+        var steps: [Step] = [
             tapStep,
             doubleTapStep,
             swipeDownStep,
             swipeHorizontallyStep,
             longPressStep,
+            touchDownStep,
             panStep,
             pinchStep,
             rotateStep,
             transformStep
         ]
 
+        if #available(iOS 9.0, *), let index = steps.index(where: { $0 === panStep }) {
+            steps.insert(forceTouchStep, at: index)
```

Comment:
Just a nit-pick but this feature is actually called 3D touch and not force-touch. Not sure if it matters, naming-wise 🤔

Reply:
I called it `forceTouch` because it reads the `force` property of the `UIEvent` property.
And var names cannot start by a digit anyway so `forceTouch` is still better than `threeDTouch` 😅